### ISSUE TITLE
Totally wrong historical misunderstang about iran(nwo)

### ISSUE THREAD

**Issue by POORIANSARI** [simsulla/nwomod#1193]
Yesterday I downloaded the new version and then realized Iran in both 1992 and 1946 does not have the 
technologies which had before in recent versions.I had played this mod for about a year and even before
I had some complaints about some other mistakes. I have made a list about the mistakes here it is
1.Iran in 1946 was not an absolute monarchy.(https://en.wikipedia.org/wiki/Iranian_Constitutional_Revolution) and it had registered voting system.So the gov type and the flag are not correct.
2.For God's sake IRAN (and also turkey) had 
3.Thanks for mentioning Abadan crisis it does not tell the accuarate story.The british sanctions must have a lot more impact on Iranian economy
4.......................................
read this please!
[read.txt](https://github.com/simsulla/nwomod/files/1800326/read.txt)

**Comment by simsulla** [comment]
Create, POORIANSARI, if you want to contribute please do! We love to have a ballanced and more accurate portrayal of all countries arund the world. In particular if you have some ideas how to make the Ummah more realistic please do (it is way to "unrealistic" at the moment).

Some notes.

On 11. I get your point. But this is difficult. We (due to mod complexity) want to keep the number of gov types low. So either Isis or Iran must not be a theocracy or we need to make the gov type theocracy fit both reasonably (but neither well).

On 14. it is not Iran specific, and one need to take care before changing it (like having some fertility lowering effect in Iran specific https://en.wikipedia.org/wiki/Family_planning_in_Iran etc.)

**Comment by POORIANSARI** [comment]
hi simsulla
I really want to thank you for your reply.I am actually eager to help but
don't know how to start.I need help for the mod creating mechanism in your
github.but about your notes
I should say it would be better to find mod editors and game programmers
from every country and ask help from them for their own country.
thanks again

**Comment by simsulla** [comment]
I agree. People with lokal knowledge tend to have a more realistic and nuanced understanding of news events. Everyone is welcome to help out with the mod.

To mod, do like this. Clone the mod from github. Do your changes, and then when you are satisfied, me or @arandomperson5000  will review and commit them to the main mod code track. The best model is to create an issue per change (i.e. you listed 16. suggested changes), and make several "Pull Requests" one per issue.
